### PR TITLE
Corrected email addresses, spacing, added SAP

### PR DESCRIPTION
**Pull request by RoAnnC1**
Charlie please review.

### CODE REVIEW

**Review by charlieconcur — APPROVED** [review]
Looks good to me...!

**Review by sarraloew — APPROVED** [review]
Thank you!

File: src/api-guides/e-receipts.markdown [hunk 1]
```diff
@@ -1,109 +1,108 @@
...
 
-**Existing App Center Partners:**  The Concur Platform team will need to configure your existing sandbox among other things to enable access.  You can make that request by contacting your [Alliance Manager](mailto:AppCenterAlliance@concur.com?subject=Receipts%20v4%20Migration%20Request).
+**Existing App Center Partners:**  The SAP Concur Platform team will need to configure your existing sandbox, among other things to enable access.  You can make that request by contacting your [Alliance Manager](mailto:concur_AppCenterAlliance@sap.com?subject=Receipts%20v4%20Migration%20Request).
 
 
 ### User Connections / Authentication
 
-The below Authentication Methods are available to obtain an access token.  Password Grant or Authorization Grant or One-Time Password Grant is required for Certification:
+;The below Authentication Methods are available to obtain an access token. Password Grant or Authorization Grant or One-Time Password Grant is required for Certification:
```
Comment (charlieconcur) [inline src/api-guides/e-receipts.markdown]: Do we need the semicolon here?

File: src/api-guides/e-receipts.markdown [hunk 2]
```diff
@@ -1,109 +1,108 @@
...
 
-- [Password Grant](/api-reference/authentication/apidoc.html#password_grant) – Connect from the Concur App Center.
-- [Authorization Grant](/api-reference/authentication/apidoc.html#auth_grant) – Connect from the Partner App.
-- [One-Time Password Grant](/api-reference/authentication/apidoc.html#otp_grant) – Connect from the Partner App utilizing Email.
+- [Password Grant](/api-reference/authentication/apidoc.html#password_grant) &ndash; Connect from the SAP Concur App Center.
```
Comment (charlieconcur) [inline src/api-guides/e-receipts.markdown]: While I agree that it probably looks a bit nicer to use the &ndash; -- in practical terms it just makes the docs harder to read and edit. I would move these back to plain dash.

File: src/api-guides/e-receipts.markdown [hunk 3]
```diff
@@ -1,109 +1,108 @@
...
 Example:   https://{partner_redirect_URI}?id=8568a4cd-8ffc-49d6-9417-be2d69aa075f&requestToken=5l85ae5a-426f-4d6f-8af4-08648c4b696b
-5. When your application receives the redirect call strip the **id** and **requestToken** values from the URI and use those on a Post request to the Concur Authorization service to obtain the official oauth2 accesstoken and refreshtoken for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being Geo Aware](#being_geo_aware).
-6. Decode the **id\_token** to obtain the **sub** value and store this value as the user **id** (see [https://jwt.io](https://jwt.io))
-7. An access token is valid only for one hour.  The access token should be cached in memory and discarded after use.
+5. When your application receives the redirect call strip the **id** and **requestToken** values from the URI and use those on a Post request to the SAP Concur Authorization service to obtain the official OAuth2 **accessToken** and **refreshToken** for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being Geo Aware](#being_geo_aware).
```
Comment (charlieconcur) [inline src/api-guides/e-receipts.markdown]: When noting key names or values use tick marks:

```
`id`
`requestToken`
`accessToken`
`refreshToken`
```

Comment (charlieconcur) [inline src/api-guides/e-receipts.markdown]: Anytime you see underscores, convert them to dashes:

```
Before: #being_geo_aware
After:  #being-geo-aware
```

File: src/api-guides/e-receipts.markdown [hunk 4]
```diff
@@ -160,14 +159,14 @@ To [revoke](/api-reference/authentication/apidoc.html#revoke_token) a user&#39;s
...
+#### SAP Concur Has Multiple Datacenters
 
 US = `https://us.api.concursolutions.com`
 
 EMEA = `https://emea.api.concursolutions.com`
 
 China = `https://cn.api.concurcdc.cn`
 
-You will need to be aware of the geolocation where the user exists in and make the call to the APIs correctly.  If there is a case where you will not / do not know the user&#39;s geolocation, then you should make the API call using the default US Base URI `https://us.api.concursolutions.com` and expect a geolocation error which will return the correct geoLocation for the user.  Sample of Error below:
+You will need to be aware of the geolocation where the user exists in and make the call to the APIs correctly. If there is a case where you will not/do not know the user&#39;s geolocation, then you should make the API call using the default US Base URI `https://us.api.concursolutions.com` and expect a geolocation error which will return the correct geoLocation for the user. Sample of Error below:
```
Comment (charlieconcur) [inline src/api-guides/e-receipts.markdown]: This: `&#39;` was leftover from someone who didn't know the difference between a tick mark and a single quote on their keyboard. Please convert all of these to single quotes when you find them.

**Review by charlieconcur — CHANGES_REQUESTED** [review]

File: src/api-guides/e-receipts.markdown (hunk 1 above)
Comment (RoAnnC1) [reply src/api-guides/e-receipts.markdown]: No, sorry. Not sure what happened there.

File: src/api-guides/e-receipts.markdown (hunk 2 above)
Comment (RoAnnC1) [reply src/api-guides/e-receipts.markdown]: Changed back to hyphen.

File: src/api-guides/e-receipts.markdown (hunk 3 above)
Comment (RoAnnC1) [reply src/api-guides/e-receipts.markdown]: Changed

Comment (RoAnnC1) [reply src/api-guides/e-receipts.markdown]: Changed

File: src/api-guides/e-receipts.markdown (hunk 4 above)
Comment (RoAnnC1) [reply src/api-guides/e-receipts.markdown]: Fixed

**Comment by RoAnnC1** [comment]
Fixed changed Charlie requested

**Pull request closed without merging**